### ISSUE TITLE
packer-post-processor-vagrant-s3 broken with Packer 1.5.1

### ISSUE THREAD

**Issue by shleimel** [lmars/packer-post-processor-vagrant-s3#58]
```2020/02/02 02:03:43 Current exe path: /usr/local/bin/packer
2020/02/02 02:03:43 Creating plugin client for path: /usr/local/bin/packer
2020/02/02 02:03:43 Starting plugin: /usr/local/bin/packer []string{"/usr/local/bin/packer", "plugin", "packer-post-processor-vagrant"}
2020/02/02 02:03:43 Waiting for RPC address for: /usr/local/bin/packer
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: [INFO] Packer version: 1.5.1 [go1.13.5 linux amd64]
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: Attempting to open config file: /user/.packerconfig
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: [WARN] Config file doesn't exist: /user/.packerconfig
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: Setting cache directory: /app/utility/deploy/packer/windows/packer_cache
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: args: []string{"packer-post-processor-vagrant"}
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: Plugin address: unix /tmp/packer-plugin662587117
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: Waiting for connection...
2020/02/02 02:03:43 Received unix RPC address for /usr/local/bin/packer: addr is /tmp/packer-plugin662587117
2020/02/02 02:03:43 packer-post-processor-vagrant plugin: Serving a plugin connection...
2020/02/02 02:03:43 Plugin could not be found at  (exec: "": executable file not found in $PATH). Checking same directory as executable.
2020/02/02 02:03:43 Current exe path: /usr/local/bin/packer
2020/02/02 02:03:43 Creating plugin client for path: /usr/local/bin
2020/02/02 02:03:43 Starting plugin: /usr/local/bin []string{"/usr/local/bin"}
2020/02/02 02:03:43 Build debug mode: false
2020/02/02 02:03:43 Force build: false
2020/02/02 02:03:43 On error:
2020/02/02 02:03:43 Waiting on builds to complete...
==> Builds finished but no artifacts were created.
2020/02/02 02:03:43 [INFO] (telemetry) Finalizing.
Failed to initialize build 'qemu': error initializing post-processor 'vagrant-s3': fork/exec /usr/local/bin: permission denied
```

Packer seems to have changed the plugin contract since 1.4.5 so an update is required.

**Comment by Jalle19** [comment]
Are you sure you have the plugin binary in the right directory?

**Comment by shleimel** [comment]
```
user@pc:/packer/windows# packer build win7-32.json
Failed to initialize build 'qemu': error initializing post-processor 'vagrant-s3': fork/exec /usr/local/bin: permission denied

==> Builds finished but no artifacts were created.
user@pc:/packer/windows# ll ~/.packer.d/plugins/
total 17676
drwxr-xr-x 2 user user     4096 Nov 10 19:14 ./
drwxr-xr-x 3 user user     4096 Feb  2 02:01 ../
-rwxr-xr-x 1 user user 18090432 Feb  2 20:21 packer-post-processor-vagrant-s3*
```

I downgraded to packer 1.4.5 and the same setup works so I assumed it packer 1.5.1 - is there something I'm missing?

**Comment by madeddie** [comment]
To make the module compatible with the latest Packer I created #59

**Comment by lmars** [comment]
@shleimel this should be fixed by #59, can you check and then close?

**Comment by Jalle19** [comment]
Assuming this is fixed
